hibernate-core/src/main/java/org/hibernate/persister/entity/AbstractPropertyMapping.java

```diff
@@ -127,6 +131,18 @@ private void logDuplicateRegistration(String path, Type existingType, Type type)
 		}
 	}
 
+	private void logIncompatibleRegistration(String path, Type existingType, Type type) {
+		if ( LOG.isTraceEnabled() ) {
+			LOG.tracev(
+					"Skipped adding same named type incompatible property to base type [{0}] for property [{1}], existing type = [{2}], incoming type = [{3}]",
```

Comment:
Can we say something like this, instead?

````
"Skipped adding attribute [{1}] to base-type [{0}] as more than one sub-type defined the attribute using incompatible types (strictly speaking the attributes are not inherited); existing type = [{2}], incoming type = [{3}]"
````

Reply:
Thanks @sebersole,
I'll change it.